lib/generators/htmx/install_generator.rb

```diff
@@ -38,7 +63,7 @@ def setup_sprockets
       end
 
       def setup_webpacker
-        `yarn add htmx.org`
+        run 'yarn add htmx.org'
```

Comment:
the #run helper is from Thor gem (thor-1.2.2/lib/thor/actions.rb
). Has some nice stuff.